src/components/PDFThumbnail/index.native.tsx

```diff
@@ -1,19 +1,25 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {View} from 'react-native';
 import Pdf from 'react-native-pdf';
 import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import addEncryptedAuthTokenToURL from '@libs/addEncryptedAuthTokenToURL';
+import variables from '@styles/variables';
 import type PDFThumbnailProps from './types';
 
-function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, enabled = true, onPassword}: PDFThumbnailProps) {
+function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, enabled = true, onPassword, onLoadError}: PDFThumbnailProps) {
     const styles = useThemeStyles();
+    const theme = useTheme();
     const sizeStyles = [styles.w100, styles.h100];
+    const [hasError, setHasError] = useState(false);
```

Comment:
`hasError` variable name doesn't look good here. Instead we can use something like `failedToLoad` @hayata-suenaga @Krishna2323

Reply:
`failedToLoad` sounds good 😄